src/main.rs

```diff
@@ -47,11 +47,31 @@ fn shebang(nightly: bool, quiet: bool) -> String {
     format!("#!/usr/bin/env {cargo_invocation}{quiet_arg}")
 }
 
-fn frontmatter() -> &'static str {
-    "---
-[dependencies]
+fn frontmatter(release_profile: bool) -> String {
+    let mut buf = String::new();
+    
+    buf.push_str("---\n");
+    buf.push_str("[dependencies]\n\n");
 
----"
+    if release_profile {
+        buf.push_str("[profile.dev]\n");
+        buf.push_str(release_profile_settings());
+    }
+
+    buf.push_str("---");
+
+    buf
+}
+
+fn release_profile_settings() -> &'static str {
+    r#"opt-level = 3
+debug = false
+debuginfo = "None"
+debug-assertions = false
+overflow-checks = false
+incremental = false
+codegen-units = 16
```

Comment:
I believe this can all be replaced with `inherits = "release"`

Reply:
seems like this is not allowed in `dev` and `release`:
```
error: `inherits` must not be specified in root profile `dev`
```